bin/v-add-web-domain-backend

```diff
@@ -55,7 +55,7 @@ fi
 
 # Allocating backend port
 backend_port=9000
-ports=$(grep -v '^;' $pool/* 2> /dev/null | grep listen | grep -o :[0-9].*)
+ports=$(grep $pool/* 2> /dev/null | grep -o :[0-9].*)
```

Comment:
If I rebuild a domain with PHP-7_4 template for example it runs:
pool="/etc/php/7.4/fpm/pool.d"
grep $pool/* 2> /dev/null | grep -o :[0-9].*

And it freezes

Reply:
Strange. I made the same PR to Vesta in December 2018. It was excepted in October 2021 and is still the newest change in both files.

https://github.com/serghey-rodin/vesta/pull/1791

Reply:
I tested it when I made it the first time but maybe something has changed.

Reply:
https://github.com/hestiacp/hestiacp/blob/559aece5dc0012c765bf8f334fd649a39056593d/func/domain.sh#L102-L131

vs 

https://github.com/serghey-rodin/vesta/blob/bba7400b8a01d462dae4b2f805866c2ac5efbf00/func/domain.sh#L85-L103

It looks like we have made some changes to it it might be the reason why it  breaks

Reply:
It looks like as VestaCP doesn't support multiple fpm version is caused the issue 

Replacing your line with 

```
ports=$(grep listen /etc/php/*/fpm/pool.d/* | grep -o :[0-9].*)
```

Seems to work. But how do we provide the port number back to the user. 

it would be easier to supply the user with a option to spefiy the port via web domain selection